File: data/multilang_comparison/program_julia/src/path.jl
include("program_julia_head.jl")
function get_path()::UInt64
   path = get(ENV, "BENCH_PATH", nothing)
   if path !== nothing
       return parse(UInt64, path)
   else
       n = rand(UInt32)
       return (UInt64(n) << 32) | UInt64(rand(UInt32))
   end
end

File: data/multilang_comparison/program_go/program_go/src/path.go
package main
import "math/rand"
import "os"
import "strconv"
var _ = rand.Intn
func getPath() uint64 {
    path := os.Getenv("BENCH_PATH")
    if path != "" {
        val, err := strconv.ParseUint(path, 10, 64)
        if err == nil {
            return val
        }
    }
    n := uint64(rand.Uint32())
    return (n << 32) | uint64(rand.Uint32())
}

